secondaryinfo-entity-row.js

```diff
@@ -37,7 +37,7 @@ customElements.whenDefined('card-tools').then(() => {
             await this._wrappedElement.shadowRoot.querySelector("hui-generic-entity-row");
             let secondaryInfoDiv = this._wrappedElement.shadowRoot.querySelector("hui-generic-entity-row").shadowRoot.querySelector(".secondary");
             if (secondaryInfoDiv && this._config.secondary_info) {
-                let text = window.cardTools.parseTemplate(this._config.secondary_info, 'Unable to parse secondary_info config');
+                let text = window.cardTools.parseTemplate(this._config.secondary_info, {entity: this._config.entity});
```

Comment:
Could be a "const". I see no need to make it "let"

Reply:
I just didn't want to change more code than necessary and kept it with "let" as it was before.